cpu/samd5x/include/candev_samd5x.h

```diff
@@ -0,0 +1,138 @@
+#ifndef CANDEV_SAMD5X_H
+#define CANDEV_SAMD5X_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "can/candev.h"
+
+#ifndef CANDEV_SAMD5X_DEFAULT_BITRATE
+/** Default bitrate */
+#define CANDEV_SAMD5X_DEFAULT_BITRATE 500000U
+#endif
+
+#ifndef CANDEV_SAMD5X_DEFAULT_SPT
+/** Default sampling-point */
+#define CANDEV_SAMD5X_DEFAULT_SPT     875
+#endif
+
+#ifndef CANDEV_SAMD5X_DEFAULT_STD_FILTER_NUM
+#define CANDEV_SAMD5X_DEFAULT_STD_FILTER_NUM	3
```

Comment:
do not use tabs 
```suggestion
#define CANDEV_SAMD5X_DEFAULT_STD_FILTER_NUM 3
```

also check the other defines